### PR TITLE
[Accessibility] Fixed a few details related to the screen reader in documentation and sidedoc

### PR DESCRIPTION
**Pull request by veler**
Related issues : [https://github.com/Microsoft/pxt/issues/2588](https://github.com/Microsoft/pxt/issues/2588), [https://github.com/Microsoft/pxt/issues/2540](https://github.com/Microsoft/pxt/issues/2540), [https://github.com/Microsoft/pxt/issues/2568](https://github.com/Microsoft/pxt/issues/2568)

### CODE REVIEW

File: webapp/src/container.tsx [hunk 1]
```diff
@@ -104,11 +104,11 @@ export class SideDocs extends data.Component<ISettingsProps, {}> {
             </button>
             <div id="sidedocs">
                 <div id="sidedocsbar">
-                    <h3><a className="ui icon link" role="link" tabIndex={0} data-content={lf("Open documentation in new tab") } aria-label={lf("Open documentation in new tab") } title={lf("Open documentation in new tab") } onClick={() => this.popOut() } >
+                    <a className="ui icon link" role="link" tabIndex={0} data-content={lf("Open documentation in new tab") } aria-label={lf("Open documentation in new tab") } title={lf("Open documentation in new tab") } onClick={() => this.popOut() } >
```
Comment (microsoftsam) [inline webapp/src/container.tsx]: Do we need both aria-label and title?

Comment (veler) [inline webapp/src/container.tsx]: Good catch

File: docfiles/toc.html [hunk 2]
```diff
@@ -6,10 +6,10 @@
             Documentation
         </a>
         <div class="item">
-            <form id="tocsearch2" method="get" target="_blank" action="https://www.bing.com/search">
+            <form id="tocsearch2" role="form" method="get" target="_blank" action="https://www.bing.com/search">
                 <div class="ui fluid icon input">
                     <input type="hidden" name="q1" value="site:@homeurl@" />
-                    <input type="search" name="q" placeholder="Search...">
+                    <input type="search" name="q" placeholder="Search..." aria-label="Type anything to search in the documentation">
```
Comment (microsoftsam) [inline docfiles/toc.html]: Simply "Search Documentation" will do.

File: docfiles/tocheader.html [hunk 3]
```diff
@@ -5,10 +5,10 @@
         Documentation
     </a>
     <div class="item">
-        <form id="tocsearch1" method="get" action="https://www.bing.com/search">
+        <form id="tocsearch1" role="form" method="get" action="https://www.bing.com/search">
             <div class="ui fluid icon input">
                 <input type="hidden" name="q1" value="site:@homeurl@" />
-                <input type="search" name="q" placeholder="Search...">
+                <input type="search" name="q" placeholder="Search..." aria-label="Type anything to search in the documentation">
```
Comment (microsoftsam) [inline docfiles/tocheader.html]: Screen readers already point out when something is an input field. Simply aria-label="Search documentation" should do.

**Comment by microsoftsam** [comment]
Branch has merge conflicts, please merge and fix the one minor issue.

File: docfiles/tocheader.html (hunk 3 above)
Comment (veler) [inline docfiles/tocheader.html]: Fixed

**Pull request merged**